src/site/content/en/lighthouse-performance/third-party-facades/index.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Lazy load third-party resources with facades
+description: |
+  Learn about the opportunities to lazy load third-party resources with facades.
+date: 2020-12-01
+updated: 2020-12-01
+web_lighthouse:
+  - third-party-facades
```

Comment:
@kaycebasques What exactly does this YAML field (`web_lighthouse`) do?

Reply:
I just added a section to explain it here: https://web.dev/handbook/yaml-front-matter/#web_lighthouse